generate_message.py

```diff
@@ -0,0 +1,15 @@
+
+
+def generate_message(diff_string,website_name):
+    message_string = """
+    Dear User,
+
+     This message is to inform you that a difference has been detected in the %s website. Please
```

Comment:
```suggestion
     A difference has been detected in the %s website. Please
```
I would also drop the `Dear User,` in the line above this because they make the message too verbose